README.md

```diff
@@ -260,6 +260,37 @@ configuration options can be updated except:
 - `maxFreeSockets`
 - `centralConfig`
 
+### `client.addMetadataFilter(fn)`
+
+Add a filter function for the ["metadata" object](https://www.elastic.co/guide/en/apm/server/current/metadata-api.html)
+sent to APM server. Here is an example of a filter that removes the
+`metadata.process.argv` field:
+
+```js
+apm.addMetadataFilter(function dropArgv(md) {
+  if (md.process && md.process.argv) {
+    delete md.process.argv
+  }
+  return md
+})
+```
+
+It is up to the user to ensure the returned object conforms to the
+[metadata schema](https://www.elastic.co/guide/en/apm/server/current/metadata-api.html),
+otherwise APM data injest will be broken. An example of that (when used with
+the Node.js APM agent) is this in the application's log:
+
+```
+[2021-04-14T22:28:35.419Z] ERROR (elastic-apm-node): APM Server transport error (400): Unexpected APM Server response
+APM Server accepted 0 events in the last request
+Error: validation error: 'metadata' required
+  Document: {"metadata":null}
+```
+
+See the [APM Agent `addMetadataFilter` documentation](https://www.elastic.co/guide/en/apm/agent/nodejs/current/agent-api.html#apm-add-metadata-filter)
+for further details.
```

Comment:
saying it out loud -- [this](https://github.com/elastic/apm-agent-nodejs/pull/2045/files#diff-23427ac409c61f7de0225443c0c7c77aca90b732e73957bad73edb357c30addeR163) will need to land before this link works.

Reply:
Yup.